aries-site/src/pages/templates/login-screen.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
```

Comment:
This file is not being used anywhere, correct?

Will this this file be implemented when work for https://github.com/hpe-design/design-system/issues/412 is done? If so, we should reference it in that issue so that this is seen and doesn't get orphaned.

Reply:
Right, this is not currently being used. I think it would be better to just remove the file for now. Then, it can be added back in when it will be displayed on the site.